src/components/folio-number-input/FolioNumberInput.vue

```diff
@@ -39,6 +38,9 @@ export default class FolioNumberInput extends Vue {
   /** Disabled prop. */
   @Prop({ default: false }) readonly disabled!: boolean
 
+  /** Local properties */
+  private folioNumberString: string
```

Comment:
Use local property to handle state changes. Otherwise we get this error:

<img width="803" alt="Screenshot 2024-01-03 at 8 37 56 PM" src="https://github.com/bcgov/bcrs-shared-components/assets/122323255/8862a5ca-2837-4c57-a355-0b1f57e7f4c3">

Reply:
Right, because you can't set `folioNumber` because it's a read-only prop.